cmd/up.go

```diff
@@ -56,5 +58,6 @@ var upCmd = &cobra.Command{
 func init() {
 	upCmd.Flags().BoolVar(&UpEmptyVols, "emptyvols", false, "Use empty volumes. Do not generate PersistentVolumeClaim")
 	upCmd.Flags().IntVar(&UpReplicas, "replicas", 1, "Specify the number of replicas generated")
+	upCmd.Flags().BoolVar(&UpInsecureRepo, "insecure-repository", false, "Specify to use insecure docker repository while generating Openshift image stream object")
```

Comment:
Hi @qujinping this description may be better:
`Use an insecure Docker repository for OpenShift ImageStream`